### PR TITLE
use uuid for generated task ids

### PR DESCRIPTION
**Pull request by kai5263499**
This is a small change to use a uuid for the task id instead of a random generator. I'm not sure if its more performant but it should provide a better assurance that the id generated would be unique. This will help if we want to emit task details for reporting purposes later.

### CODE REVIEW

File: crontainer/task.go [hunk 1]
```diff
@@ -43,13 +43,3 @@ func (t *Task) Run() {
 func (t *Task) log(text interface{}) {
 	log.Printf("[%15s] %s", t.Name, text)
```
Comment (neckhair) [inline crontainer/task.go]: Could you shorten this to 8 characters? That would be the first character block before the hyphen. For the output that should be enough.
```go
t.Name[:8]
```

File: crontainer/task.go [hunk 2]
```diff
@@ -14,7 +14,7 @@ type Task struct {
 
 func NewTask(command string, schedule string, name string) *Task {
 	if name == "" {
-		name = randStringBytes(15)
+		name = uuid.NewV4()
```
Comment (neckhair) [inline crontainer/task.go]: This will not work. `name` is a `string`, but `New4()` returns a `UUID`. You can write `uuid.NewV4().String()`.

**Review by neckhair — CHANGES_REQUESTED** [review]
Thanks for this PR. That's a good idea. Don't worry about performance. It's only loaded once on startup. That shouldn't have any impact.
CI seems to be broken. I'll try to fix that now.

**Comment by neckhair** [comment]
`.travis.yml` should be fixed now. Could you rebase your branch on master?

**Comment by kai5263499** [comment]
Thanks for the feedback! I've rebased onto your master branch, used the .String() method of uuid, and cut the characters down to 8

**Review by neckhair — APPROVED** [review]
